pom.xml

```diff
@@ -91,18 +90,6 @@
             <artifactId>aws-java-sdk-dynamodb</artifactId>
             <version>1.11.718</version>
         </dependency>
-        <dependency>
```

Comment:
Why was this deleted? I believe only the duplicate should be removed.

Reply:
it is duplicated, check the first dependency :)

Reply:
There it has scope test, not compile